src/Moryx.Model.Sqlite/SqliteModelConfigurator.cs

```diff
@@ -68,7 +69,9 @@ public override DbContextOptions BuildDbContextOptions(IDatabaseConfig config)
 
         private static string BuildConnectionString(IDatabaseConfig config)
         {
-                return config.ConnectionSettings.ConnectionString;
+            var connectionStringBuilder = new SqliteConnectionStringBuilder(config.ConnectionSettings.ConnectionString);
+            connectionStringBuilder.Pooling = false;
```

Comment:
As mentioned above, why not doing this only for the case of deleting a database?

Reply:
I agree, is there a reason I can't see? 🙈

Reply:
we are using the File.Remove() to delete the database. we are not using a query like we do in Postgres to delete the database therefore we don't have access to SqlConnection. refer to [sqlite delete drop database ](https://www.tutlane.com/tutorial/sqlite/sqlite-delete-drop-database#:~:text=Like%20most%20relational%20database%20systems,and%20it%20is%20case%2Dsensitive.)

Reply:
We might rethink/rework how to create/delete databases, or which conditions should apply to consider a database being created/deleted.